### PR TITLE
[WIP] First pass on ROM integration

### PR DESCRIPTION
**Pull request by sapitts**
for discussion later, here's my work in progress (no docs, uncleaned code) on the ROM integration in a non-AD version @bwspenc @dschwen @tophmatthews

### CODE REVIEW

File: src/vpscrom/SS316ROMPrediction.C [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#include "SS316ROMPrediction.h"
+
+SS316ROMPrediction::SS316ROMPrediction(){};
+
+void
+SS316ROMPrediction::computeROMPredictions(const double & dt,
```
Comment (tophmatthews) [inline src/vpscrom/SS316ROMPrediction.C]: Was this file from LANL originally? I received similar functions that I used as a starting point

Comment (sapitts) [reply src/vpscrom/SS316ROMPrediction.C]: No, this was my work that I shared with Arul and PA (Kairos)via my blackbear branch when they met with me in July

Comment (sapitts) [reply src/vpscrom/SS316ROMPrediction.C]: From LANL I received a FORTRAN standalone version of the ROM and later a cpp version (from Aaron). I've tried to denote with comments the LANL standalone cpp version of the code

Comment (tophmatthews) [reply src/vpscrom/SS316ROMPrediction.C]: Gotcha! Just wanted to make sure you got fair credit for all your work.

**Comment by bwspenc** [comment]
@sapitts I think this code is all out of date now, isn't it?  My understanding is that instead of this, we want to move over the ROM that is currently in the TM test area here.

**Comment by sapitts** [comment]
oh yep, this is out of date code. I will close this now

**Pull request closed without merging**